cmd/neofs-adm/internal/modules/morph/verified_domains.go

```diff
@@ -0,0 +1,219 @@
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient/invoker"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/vm/emit"
+	nnsrpc "github.com/nspcc-dev/neofs-contract/rpc/nns"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-adm/internal/modules/config"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func verifiedNodesDomainAccessList(cmd *cobra.Command, _ []string) error {
+	vpr := viper.GetViper()
+
+	n3Client, err := getN3Client(vpr)
+	if err != nil {
+		return fmt.Errorf("open connection: %w", err)
+	}
+
+	nnsContractAddr, err := nnsrpc.InferHash(n3Client)
+	if err != nil {
+		return fmt.Errorf("get NeoFS NNS contract address: %w", err)
+	}
+
+	domain := vpr.GetString(domainFlag)
+
+	err = iterateNNSDomainTextRecords(invoker.New(n3Client, nil), nnsContractAddr, domain, func(rec string) error {
+		cmd.Println(rec)
+		return nil
+	})
+	if err != nil {
+		switch {
+		default:
+			return fmt.Errorf("handle domain %q records: %w", domain, err)
+		case errors.Is(err, errDomainNotFound):
+			cmd.Println("Domain not found.")
+			return nil
+		case errors.Is(err, errMissingDomainRecords):
+			cmd.Println("List is empty.")
+			return nil
+		}
+	}
+
+	return nil
+}
+
+func verifiedNodesDomainSetAccessList(cmd *cobra.Command, _ []string) error {
+	vpr := viper.GetViper()
+
+	strNeoAddresses := vpr.GetStringSlice(neoAddressesFlag)
+	strPublicKeys := vpr.GetStringSlice(publicKeysFlag)
+	if len(strNeoAddresses)+len(strPublicKeys) == 0 {
+		// Track https://github.com/nspcc-dev/neofs-node/issues/2595.
+		return errors.New("neither Neo addresses nor public keys are set")
+	}
+
+	if len(strNeoAddresses)*len(strPublicKeys) != 0 {
+		// just to make sure
+		panic("mutually exclusive flags bypassed Cobra")
+	}
+
+	var err error
+	var additionalRecords []string
+
+	if len(strNeoAddresses) > 0 {
+		for i := range strNeoAddresses {
+			for j := i + 1; j < len(strNeoAddresses); j++ {
+				if strNeoAddresses[i] == strNeoAddresses[j] {
+					return fmt.Errorf("duplicated Neo address %s", strNeoAddresses[i])
+				}
+			}
+
+			_, err = address.StringToUint160(strNeoAddresses[i])
+			if err != nil {
+				return fmt.Errorf("address #%d is invalid: %w", i, err)
+			}
+		}
+
+		additionalRecords = strNeoAddresses
+	} else {
+		additionalRecords = make([]string, len(strPublicKeys))
+
+		for i := range strPublicKeys {
+			bKey, err := hex.DecodeString(strPublicKeys[i])
+			if err != nil {
+				return fmt.Errorf("public key #%d is not a HEX string: %w", i, err)
+			}
+
+			for j := i + 1; j < len(strPublicKeys); j++ {
+				if strPublicKeys[i] == strPublicKeys[j] {
+					return fmt.Errorf("duplicated public key %s", strPublicKeys[i])
+				}
+			}
+
+			buf := io.NewBufBinWriter()
+			emit.CheckSig(buf.BinWriter, bKey)
+
+			additionalRecords[i] = address.Uint160ToString(hash.Hash160(buf.Bytes()))
+		}
+	}
+
+	walletDir := config.ResolveHomePath(vpr.GetString(alphabetWalletsFlag))
+
+	wallets, err := openAlphabetWallets(vpr, walletDir)
```

Comment:
This particular command doesn't require alphabet wallets.

Reply:
changed to work with any account